actionpack/lib/action_dispatch/railtie.rb

```diff
@@ -13,12 +13,12 @@ class Railtie < Rails::Railtie # :nodoc:
     config.action_dispatch.default_charset = nil
     config.action_dispatch.rack_cache = false
     config.action_dispatch.http_auth_salt = "http authentication"
-    config.action_dispatch.signed_cookie_salt = "signed cookie"
-    config.action_dispatch.encrypted_cookie_salt = "encrypted cookie"
-    config.action_dispatch.encrypted_signed_cookie_salt = "signed encrypted cookie"
     config.action_dispatch.use_authenticated_cookie_encryption = false
     config.action_dispatch.perform_deep_munge = true
 
+    config.action_dispatch.legacy_encryptor_schemes = []
+    config.action_dispatch.legacy_verifier_schemes  = []
```

Comment:
I like the idea of passing an encryptor or verifier directly. This would be for more advanced uses and would be great to support.

Under the hood, I think we can just have these instances be passed to `KeyRotator` and have the class able to insert arbitrary encryptors or verifiers into its chain. Modifying `KeyRotator` to support this should somewhat address @bdewater's feedback about making `KeyRotator` more versatile in general.

I also think the `config.secrets.rotate` syntax should be clear enough. I feel the "rotating out" concept is implied by the feature as a whole.